app/forms/two_factor_authentication/phone_deletion_form.rb

```diff
@@ -0,0 +1,30 @@
+module TwoFactorAuthentication
+  class PhoneDeletionForm
+    attr_reader :user, :configuration
+
+    def initialize(user, configuration)
+      @user = user
+      @configuration = configuration
+    end
+
+    def submit
+      success = configuration_absent? || multiple_factors_enabled? && configuration_destroyed
+
+      FormResponse.new(success: success, errors: {}, extra: {})
```

Comment:
One drawback of making the `errors` hash empty is that if this fails we won't be able to tell from the logs why it failed.

Reply:
Yeah. I was wondering what to put in there. I'll add some information for logging. I also need to update text and make some other tweaks based on a demo for John D. yesterday.

Reply:
Neato, looks addressed in the latest changeset.